editors/vscode/syntaxes/jakt.tmLanguage.json

```diff
@@ -784,8 +784,16 @@
           "match": "\\b(as)\\b\\s"
         },
         {
-          "name": "keyword.control.other.jakt",
-          "match": "\\b(import)\\b"
+          "name": "keyword.control.import.jakt",
+          "match": "\\b(import)(?:\\s*)(?:(?<=\\s+)(?:(extern\\s+c)|(extern)))?\\b",
```

Comment:
Why the unbounded backwards assertion? that's technically not valid in ecma262
```
\b(import)(?:\s*)(?:(\bextern\s+c)|(\bextern))?\b
```
sounds just fine

Reply:
Changed to your propsal :)